### DISCUSSION TITLE
Downloading the program?

### DISCUSSION THREAD

**Discussion by rougeboijared** [CommandPost/CommandPost#2909] [Technical Support]
I'm new and don't understand how to download the program. I'm I missing something? I see a lot of videos on YouTube but I don't see one about downloading them program. I hope this isn't a stupid question and if so I'm stupid for asking but I need help. Can anybody help a brother out?

**Comment by latenitefilms** [comment]
Go to:

https://commandpost.io

…and follow the links.

**Reply by rougeboijared** [reply]
Is it the [CommandPost_1.3.4.dmg](https://github.com/CommandPost/CommandPost/releases/download/1.3.4/CommandPost_1.3.4.dmg)

**Reply by latenitefilms** [reply]
Yes, correct.

**Reply by rougeboijared** [reply]
So I downloaded the program I guess I need help making it work.

**Reply by latenitefilms** [reply]
What issues are you having? Feel free to post here, or there’s quite a few users in our Facebook Group that might also be able to help too.

**Reply by rougeboijared** [reply]
https://user-images.githubusercontent.com/103217266/162556656-2707bd93-f354-40ef-939f-9d0251b106e7.MOV


What do I do now because it's not working and I have commandpost downloaded?

**Reply by latenitefilms** [reply]
Have you enabled Monogram support in the Control Surface preferences within CommandPost? It looks like maybe you’re using a Final Cut Pro layout in Monogram Creator rather than CommandPost?
